chore(blog): add mascot blog (#3733)
asyncapi/website: markdown/blog/asyncapi-mascot-redesign.md

---

> title: AsyncAPI Mascot Redesign: The Biography of Eve and Chan

Reviewer: ```suggestion
title: AsyncAPI Mascot Redesign - The Biography of Eve and Chan
```

> date: 2025-02-25T06:00:00+01:00

Reviewer 2: ```suggestion
date: 2025-03-24T06:00:00+01:00
```

type: Community
tags:
  - Mascot
  - Design
  - Marketing
  - Community
cover: /img/posts/mascot-banner.webp
authors:
  - name: Victoria Nduka
    photo: /img/avatars/nwanduka.webp
    link: https://www.linkedin.com/in/victorianduka/
    byline: User Experience Designer
excerpt: 'The story behind Eve and Chan—how they started, why they changed, and where they’re headed next.'
---

# AsyncAPI Mascot Redesign: The Biography of Eve and Chan

- [The Birth of Eve and Chan](#the-birth-of-eve-and-chan)
- [Why the Redesign?](#why-the-redesign)
- [Meet Eve and Chan](#meet-eve-and-chan)
- [What’s Next for Eve and Chan?](#whats-next-for-eve-and-chan)

> AsyncAPI’s mascots, Eve and Chan, have had a brand refresh, and I’m proud to say that I had a hand in their redesign. Now, seeing the community use them in different ways just warms my heart. Any designer will tell you that nothing beats the feeling of seeing your work come to life beyond the Figma canvas, and that’s exactly how I feel about the new mascots.

Reviewer 2: ```suggestion
AsyncAPI’s mascots, Eve and Chan, have had a brand refresh, and I’m proud to say that I helped redesign them. Seeing the community use them in different ways warms my heart. Any designer will tell you that nothing beats seeing your work come to life beyond the Figma canvas, and that’s exactly how I feel about the new mascots.
```

> Throughout the redesign process, I kept asking myself questions to ensure the new look stayed true to their original purpose. Whose idea was it to have the mascots? Why the names Eve and Chan? Who designed them originally? Who are Eve and Chan, really?

Reviewer 2: ```suggestion
Throughout the redesign process, I kept asking myself questions to ensure the new look stayed true to its original purpose. Whose idea was it to have the mascots? Why the names Eve and Chan? Who designed them originally? Who are Eve and Chan?
```

> As it turns out, I’m not the only one with these questions. Within and outside the AsyncAPI community, folks have often wondered about the origins of these characters. So, I recently sat down with Fran Mendez, the founder of AsyncAPI, to get their backstory. This is the story of Eve and Chan—their biography, if you will.

Reviewer 2: ```suggestion
As it turns out, I’m not the only one with these questions. Within and outside the AsyncAPI community, folks have often wondered about the origins of these characters. So, I recently sat down with Fran Mendez, the founder of AsyncAPI, to get their backstory. This is the story of Eve and Chan—their biography.
```

## The Birth of Eve and Chan

> It was Fran’s idea to have a mascot that people could easily associate with the AsyncAPI brand. Most open-source projects and companies use mascots to reinforce their brand identity, so why not AsyncAPI? You know how Linux has Tux the penguin, and GitHub has Octocat? Fran wanted something like that for AsyncAPI.

Reviewer 2: ```suggestion
[Fran Méndez’s](https://www.linkedin.com/in/fmvilas) idea was to have a mascot that people could easily associate with the AsyncAPI brand. Most open-source projects and companies use mascots to reinforce their brand identity, so why not AsyncAPI? Since Linux has Tux, the Penguin, and GitHub has Octocat, Fran wanted something like that for AsyncAPI.
```

> Originally, he envisioned just one character, something eye-catching that could also work well on merch. The goal was to create swag people would actually want to buy. Not everyone would wear a hoodie with just a logo slapped on it. But a mascot? That’s stylish, versatile, and could be worn anywhere.

Reviewer 2: ```suggestion
Initially, he envisioned just one eye-catching character that could work well on merch. The goal was to create swag people would want to buy. Not everyone would wear a hoodie with just a logo slapped on it. But a mascot? That’s stylish, versatile, and could be worn anywhere.
```

> Ultimately, the mascot was to serve two purposes: strengthen AsyncAPI’s brand identity and help raise funds for the project through merch sales.

Reviewer 2: ```suggestion
Ultimately, the mascot served two purposes: strengthening AsyncAPI’s brand identity and helping raise funds for the project through merch sales.
```

> The first version of the mascots was designed by an illustrator from a studio in Colombia. He came up with two characters: a bold female lead and a quirky extraterrestrial. Now the mascots were ready—but what’s a mascot without a name?

Reviewer 2: ```suggestion
An illustrator from a studio in Colombia designed the first version of the mascots. He created two characters: a bold female lead and a quirky extraterrestrial. Now the mascots were ready—but what’s a mascot without a name?
```

> Fran took to his socials and asked the community for suggestions. Someone came up with Eve and Chan—Eve for "events" and Chan for "channels." Not only were they catchy, but they actually tied back to what AsyncAPI does. Real names with real meaning. It was brilliant.

Reviewer 2: ```suggestion
Fran took to his socials and asked the community for suggestions. Someone came up with Eve and Chan—Eve for "events" and Chan for "channels." Not only were they catchy, but they also tied back to what AsyncAPI does. Real names with real meaning. It was brilliant.
```

![Poll for mascot name suggestions][poll]

> If you’re wondering, like I did, why there are two mascots instead of just one, it’s because they were designed to tell a story. Eve is the skilled, knowledgeable one who always does things the right way, while Chan is a bit clumsy, representing the mistakes people often make in event-driven architectures.

Reviewer 2: ```suggestion
If you’re wondering, like I was, why there are two mascots instead of just one, it’s because they were designed to tell a story. Eve is the skilled, knowledgeable one who always does things correctly, while Chan is slightly clumsy. Together, they represent the mistakes people often make in event-driven architecture.
```

Author: I'll edit the file to incorporate some of the suggested changes. However, the last part—"together, they represent the mistakes..."—alters the intended meaning of the passage. Fran explained that Chan (not Eve and Chan together) represented the mistakes people make in event-driven architecture. It’s important to retain this distinction, so I’d like to adjust the wording accordingly.

Reviewer 2: Sure please go ahead. Thanks for educating me😅

Author: I already edited it. Check [my latest commit](https://github.com/asyncapi/website/pull/3733/commits/befc08a6ef1578d3e15860799d80937be8cac8d0)

## Why the Redesign?

> About two years after their debut in 2019, the mascots were due for a visual refresh. While the original designs were charming, they were also too detailed for AsyncAPI’s needs. The goal was to simplify their look—using basic shapes, fewer curves, and minimal shading—to make them more versatile and easier to use across different formats. This redesign aimed to create a cleaner, more adaptable style while preserving the essence of the original concept.

Reviewer 2: ```suggestion
About two years after their debut in 2019, the mascots were due for a visual refresh. While the original designs were charming, they were too detailed for AsyncAPI’s needs. The goal was to simplify their look—using basic shapes, fewer curves, and minimal shading—to make them more versatile and easier to use across different formats. This redesign aimed to create a cleaner, more adaptable style while preserving the essence of the original concept.
```

In 2021, a GitHub issue was created to kick off discussions and track progress on the redesign. However, for a couple of years, not much was done. Then, in September last year, I expressed interest in working on the project and was assigned the task. And the rest, as they say, is history. For more details about the redesign, you can check out the [mascot documentation here](https://github.com/asyncapi/brand/blob/e17f6bb7f877d9ae28bb9014a0f9763544f73350/illustrations/mascots/README.md).

## Meet Eve and Chan

After four months of multiple design iterations, community voting, plenty of back-and-forth, and a few unavoidable breaks, I’m excited to introduce the redesigned AsyncAPI mascots.

![AsyncAPI Mascots][mascots]

## What's Next for Eve and Chan?

Designing Eve and Chan was both a challenge and an honor, and I can’t wait to see all the fun and creative ways the community brings them to life. You’ve probably already seen them in some of our social media posts, and they’re now the face of AsyncAPI’s GitHub bots. We’re looking to feature them on swag, in documentation, at community events, and more.

> So, if you’ve ever wondered about the story behind these little characters, now you know. And if you’re part of the AsyncAPI community, get ready. You’ll be seeing a lot more of them.

Reviewer 2: ```suggestion
So, if you’ve ever wondered about the story behind these little characters, now you know, and if you’re part of the AsyncAPI community, get ready. You’ll be seeing a lot more of them.
```
